### ISSUE TITLE
Add support for maxFontSizeMultiplier Text prop

### ISSUE THREAD

**Issue by jacquesdev** [RamProg/react-native-text-link#2]
Would it be possible to support this prop https://reactnative.dev/docs/text#maxfontsizemultiplier? Right now if I resize my app all the elements that support it stay the same, but unfortunately this component does not (yet).

**Comment by RamProg** [comment]
I just published version 1.0.1 which should allow you to pass any text prop to the component, let me know how it goes.

**Comment by jacquesdev** [comment]
@RamProg - would you just pass it in as a prop under the main TextLink component, or does it need to go into an existing prop? I passed it in at the top level, but that doesn't seem to work on my side

**Comment by RamProg** [comment]
yeah, you should be able to pass any text prop inside the < TextLink /> component and it will pass it along to the < ParsedText /> dependency, make sure you updated to 1.0.1. If that doesn't work I'm afraid it won't be an easy fix because Parsed Text hasn't been updated in 3 years, so I don't think they will be adding support for anything new. Feel free to create a PR if you find a solution and I'm happy to review.
